src/emu/common/src/language.cpp

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright (c) 2018 EKA2L1 Team.
+ *
+ * This file is part of EKA2L1 project
+ * (see bentokun.github.com/EKA2L1).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+#include <common/language.h>
+#include <map>
+
+static std::map<int, std::string> lang_map {
```

Comment:
Did you write a python script for this :p. I think real language name would be more friendly for userend

Reply:
Nope, just some multi-cursor editing in VSCode. Do you know where I can find official mapping from code to displaied language name?

Reply:
I do find some of them [here](http://devlib.symbian.slions.net/s3/GUID-31C133DE-F245-5992-9A41-20A99291E72A.html#GUID-1F01A43F-64F7-5C90-92D1-974356A6C386), some languages are missing though

Reply:
just removing ELang prefix in enum names looks fine

Reply:
Yeah, that should be viable